kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/main/java/org/kie/server/remote/rest/jbpm/JbpmRestApplicationComponentsService.java

```diff
@@ -85,12 +102,22 @@
         UserTaskServiceBase userTaskServiceBase = new UserTaskServiceBase(userTaskService, context);
         RuntimeDataServiceBase runtimeDataServiceBase = new RuntimeDataServiceBase(runtimeDataService, context);
         ExecutorServiceBase executorServiceBase = new ExecutorServiceBase(executorService, context);
+        
+        FormServiceBase formServiceBase = new FormServiceBase(definitionService, deploymentService, runtimeDataService, userTaskService, context);
+        Set<FormProvider> formProviders = new HashSet<FormProvider>();
+        
+        InMemoryFormSkeletonProvider iprovider = new InMemoryFormSkeletonProvider();
```

Comment:
would make sense to use ServiceLoader to find form providers so we are not hardcoding it and allow others to include additional providers if needed